### PR TITLE
Revisit vnu-jar suppressions and fix the script to work on 32-bit OS

### PR DESCRIPTION
**Pull request by XhmikosR**
Opening this so that we can tackle any redundant suppressions, with @patrickhlauke

### CODE REVIEW

**Review by mdo — APPROVED** [review]

File: js/tests/visual/tooltip.html [hunk 1]
```diff
@@ -46,9 +46,6 @@
         <button type="button" class="btn btn-secondary" data-toggle="tooltip" data-html="true" title="<em>Tooltip</em> <u>with</u> <b>HTML</b>">
           Tooltip with HTML
         </button>
-        <svg width="30" height="20">
```
Comment (Johann-S) [inline js/tests/visual/tooltip.html]: This SVG is present because it's a visual unit test

Comment (XhmikosR) [reply js/tests/visual/tooltip.html]: Which is what I had told you the other day... we need to make this work properly and not on the circle element.

Comment (Johann-S) [reply js/tests/visual/tooltip.html]: I know but by doing that we remove a unit test previously added since a long time

Comment (XhmikosR) [reply js/tests/visual/tooltip.html]: Yes, that is why I say, we need to make sure things work with valid stuff and not invalid, random code...

Comment (Johann-S) [reply js/tests/visual/tooltip.html]: Can you open an issue to track that please ?

Comment (XhmikosR) [reply js/tests/visual/tooltip.html]: I checked the git log and the original example was added to test an invalid code from the beginning.

So, I'm not sure what to say on the issue, so feel free to make one.

**Comment by patrickhlauke** [comment]
a labour of love, this. thanks @XhmikosR for patiently coaching me through this collaborative work

**Pull request merged**